annotations/mast-fish.md

```diff
@@ -0,0 +1,24 @@
+**_mast fish_** (English); _mastefisk_ (Danish); _Mastfisch_ (German)
+
+_**klofi** m., pl. klofar_ (Old Norse) [citations: [prose](https://onp.ku.dk/onp/onp.php?o44117)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=45381)]
+
+  A piece of wood that was placed on the keelson (ON _kerling_) to keep the mast upright and absorb stress while the ship was under sail. The mast fish was typically ten feet long and faced aft to help while the mast (ON _siglutré_, _sigla_) was raised (National Geographic).
+
+<div align="center">
+  
+  ![mast fish from Gokstad ship](../images/Kerling_00_Gokstad.png)  
+  Mast fish from the Gokstad ship (Nicolaysen Pl. I, Fig. 4). The mast fish sits above the keelson.
+
+</div>
+ 
+This feature is named for its fish-like shape, though it is sometimes also called the mast support or mast-partner (Nicolaysen).
+
+---
+
+  Nicolaysen, N. (1882). Langskibet fra Gokstad ved Sandefjord. Kristiania.
+
+  Paasche, Knut. "The Tune Viking Ship Reconsidered." _International Journal of Nautical Archaeology_ 49, no. 1 (2020): 29-48. https://doi.org/10.1111/1095-9270.12391.
```

Comment:
This and the Unger are not cited here. Are they needed?

Reply:
I had kept these from the previous version of this annotation, but you're right--they aren't cited anywhere. I'll delete them.